bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -0,0 +1,672 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2021 Francesco Pannarale
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# =============================================================================
+# Preamble
+# =============================================================================
+
+"""
+Determine efficiency and exlucion distances of a PyGRB run.
+"""
+
+import sys
+import os
+import logging
+import matplotlib.pyplot as plt
+from matplotlib import rc
+import numpy as np
+import scipy
+from scipy import stats
+import pycbc.version
+from pycbc.detector import Detector
+from pycbc import init_logging
+from pycbc.results import save_fig_with_metadata
+from pycbc.results import pygrb_postprocessing_utils as ppu
+from ligo.lw import lsctables
+
+plt.switch_backend('Agg')
+rc("image")
+
+__author__ = "Francesco Pannarale <francesco.pannarale@ligo.org>"
+__version__ = pycbc.version.git_verbose_msg
+__date__ = pycbc.version.date
+__program__ = "pycbc_pygrb_efficiency"
+
+
+# =============================================================================
+# Function to calculate the error bars and fraction of recovered injections
+# (used for efficiency/distance plots)
+# =============================================================================
```

Comment:
I would rather put this kind of descriptive comment into the function's docstring, so it ends up in the documentation.